lib/generators/encoder/encoder_generator.rb

```diff
@@ -0,0 +1,25 @@
+require 'rails/generators'
+
+module Encoder
+  module Generators
+    class EncoderGenerator < Rails::Generators::NamedBase
+      source_root File.expand_path('../templates', __FILE__)
+      check_class_collision :suffix => 'Encoder'
```

Comment:
__[rubocop]__
* Style/HashSyntax: Use the new Ruby 1.9 hash syntax. [:ref](https://github.com/bbatsov/ruby-style-guide#hash-literals)